cactusbot/services/beam/handler.py

```diff
@@ -79,17 +77,23 @@ def __init__(self, channel):
     async def handle_liveloading(self, packet):
         """Handle liveloading packets."""
 
+        packet = json.loads(packet)
+
         data = packet.get("data")
-        if not isinstance(data, list) or not isinstance(data[0], str):
+        if not isinstance(data, dict):
+            return
+
+        event = data["channel"].split(":")[-1:][0]
```

Comment:
To go along with `user`/`channel` usage, we should just do:
```python
event = data["channel"].split(':')
```

(Quick note, if we could disregard this, it'd be better to just do `[-1]`, not `[-1:][0]`. :smile:)